src/components/ReportingConfig/EmailDeliveryMethodForm.jsx

```diff
@@ -10,15 +12,13 @@ const EmailDeliveryMethodForm = ({ invalidFields, config, handleBlur }) => {
   return (
     <div className="row">
       <div className="col">
-        <ValidationFormGroup
-          for="email"
-          helpText="The email(s), one per line, where the report should be sent"
-          invalidMessage="Required. One email per line. Emails must be formatted properly (email@domain.com)"
-          invalid={invalidFields.emailRaw}
+        <Form.Group
+          controlId="email"
+          isInvalid={invalidFields.emailRaw}
         >
-          <label htmlFor="email">Email(s)</label>
-          <Input
-            type="textarea"
+          <Form.Label htmlFor="email">Email(s)</Form.Label>
+          <Form.Control
+            as="textarea"
```

Comment:
We could use the `autoResize` prop here to get the textarea to autosize based on its input from the user (e.g., if there's more emails than the textarea can display at once, the textarea control will grow vertically to fit the contents which is a nice usability improvement).

See https://paragon-openedx.netlify.app/components/form/form-control/#textarea-autoresize for more details.